src/styles/index.scss

```diff
@@ -1,3 +1,90 @@
+@keyframes rotates {
+  from {
+    rotate: 0;
+  }
+
+  to {
+    rotate: 360deg;
+  }
+}
+
 body {
+  width: 100%;
+  height: 100%;
   margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  position: absolute;
+  inset: 0;
+  margin: auto;
+
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minute-hand {
+    margin: auto;
+    position: absolute;
+    left: inherit;
```

Comment:
It seems like you have used `left: inherit;` which might not work as expected because the parent element does not have a `left` property defined. You might want to give it a specific value or remove it if it's not necessary.